src/main/java/uc/user/logout/package-info.java

```diff
@@ -1 +1,4 @@
-package uc.user.logout;
\ No newline at end of file
+package uc.user.logout;
+/**
+ * Contains use case layer classes and interfaces.
```

Comment:
Would be good to specify what the logout use case specifically does and then specify the layer using the custom @layer tag